examples/dh-5-example/src/main/entity-config/exp-final-entity-options.xml

```diff
@@ -3,6 +3,8 @@
   <search:constraint name="Collection">
     <search:collection>
       <search:facet-option>limit=25</search:facet-option>
+      <search:facet-option>frequency-order</search:facet-option>
```

Comment:
These were automatically added when I deployed the app based on the latest on develop, so checking them in.